src/views/Clients/ViewClients/index.jsx

```diff
@@ -201,4 +228,4 @@ export default class ViewClients extends PureComponent {
       </Dashboard>
     );
   }
-}
+}
```

Comment:
`handleClientSearchSubmit` seems to be defined two times. Remove the old implementation.